#custom_joystick_command:
#    ros__parameters:
# removed for composition node    
#(ref: https://github.com/ros2/rclcpp/issues/715  )

        topics:
            in_joy: "/joy"
            in_mode: "/autonomous_driver"
            out_cmd: "/hw/manual_cmd"
            
        in_msg_max_period: 5000 # [milliseconds] if msg have period greater than this the car is forced to a neutral actuator position
        command_rate: 1 # [times] how many joy times it sends command msg
        mode_rate: 10 # [times] how many joy times it sends mode msg

        throttle_axis:
            index: 4
            scale: 1.0
        steering_axis:
            index: 3
            scale: 1.0
        buttons:
            neutral_mode_index: 1
            manual_mode_index: 5
            autonomous_mode_index: 0
